src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/JCloudsProvider.java

```diff
@@ -444,4 +442,21 @@ private Set<NodeCandidate> getPaidNodeCandidate(Infrastructure infra, String reg
         }).collect(Collectors.toSet());
         return result;
     }
+
+    /**
+     * @return The Operating system family based on the infrastructure type, created to patch the limitation caused
+     * by Jclouds for OpenStack (not taking the distro_family into account).
+     */
+    private String getOperatingSystemFamily(org.jclouds.compute.domain.Image image, String infrastructureType) {
```

Comment:
It is better to move the attribute type to the class header: import org.jclouds.compute.domain.Image;

Reply:
You did it in the Openstacktil.java class

Reply:
we have to Image classes org.ow2.proactive.connector.iaas.model.Image and org.jclouds.compute.domain.Image. In JCloudsProvider both Image classes are used while in Openstackutil only org.jclouds.compute.domain.Image is used

Reply:
Ah Ok. Ignore my comments then.